sqlite/migrate.go

```diff
@@ -53,3 +59,14 @@ const createMessagesTable = `CREATE TABLE IF NOT EXISTS messages (
 
 //flavor:sqlite3
 const createIndexOnProviderID = `CREATE INDEX IF NOT EXISTS idx_provider_id ON messages (provider_id);`
+
+// hermes.App migration
+const createAppsTable = `CREATE TABLE IF NOT EXISTS apps (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	name TEXT NOT NULL,
+	token TEXT NOT NULL,
+	sender TEXT NOT NULL,
+	created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	updated_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	message_count INTEGER NOT NULL DEFAULT 0
```

Comment:
i don't think that this column would be useful cause I was suggesting that each message row must have corresponding app reference, with that we can `message_counts` from messages table by counting `cost` related to `app_id` .

Reply:
@iradukunda1 this is the first iteration, I wanted to make things as simple as possible. You'll see that the tokens are not even hashed.

Reply:
okay